### ISSUE TITLE
Currency view is broken when a rate is on database

### ISSUE THREAD

**Issue by dcrocco** [odoo/odoo#30840]
Impacted versions:
12.0

Steps to reproduce:
Create a rate for a currency

Current behavior:
Compute field breaks when entering in currencies view

![image](https://user-images.githubusercontent.com/7767727/52278561-15123b80-2936-11e9-878a-77a8a4734c10.png)

**Comment by pedrobaeza** [comment]
Hi, thanks for reporting. Are you able to reproduce the problem in runbot infrastructure (http://runbot.odoo.com/runbot - admin/admin)?

If so, please explain with all the possible detailed steps to get to the error. Not having them, it's very difficult to hunt down the problem. Please don't be shy of adding details, and even better if you provide some screenshots or videos.

If not, this can be due to a configuration change, a custom/community module you have deployed in your DB or an outdated code. Please report back if it's the case.

For keeping a clean house in this issue tracker, if you don't answer in several days, this will be closed assuming it was one of the stated things.

**Comment by sswapnesh** [comment]
Hello @dcrocco  
I am not able to reproduce same on Latest Runbot v12 (community)

It looks like it is trying to convert `datetime` field while `date` field is already` date type`
https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/models/res_currency.py#L39

Did you changed field's type manually ?

**Comment by dcrocco** [comment]
Hi, no, i couldn't reproduce in Runbot.

I made a fresh install with no modules at all, just configured multi currency (The databese is configured in spanish / AR if it helps).

I looked in the db to see what may be the problem

![image](https://user-images.githubusercontent.com/7767727/52416216-df9d5780-2ac7-11e9-9091-fd71ed697311.png)

Seems that the default name of res_currency_rate name is taking datetime instead of date.

Then i tried this
`>>> from odoo import fields
>>> fields.Date.today()
datetime.date(2019, 2, 7)
`
And saw that fields.Date.today() is returning a datetime object instead of time.

Not really what to debug from now on and how to reproduce it in runbot.
